blobfuse/blobfuse.cpp

```diff
@@ -644,6 +648,50 @@ void set_up_callbacks()
     signal(SIGUSR1, sig_usr_handler);
 }
 
+/*
+ *  Check if the given directory is already mounted or not
+ *  If mounted then re-mounting again shall fail.
+ */ 
+bool is_directory_mounted(const char* mntDir) {
+     struct mntent *mnt_ent;
+     bool found = false;
+     FILE *mnt_list;
+ 
+     mnt_list = setmntent(_PATH_MOUNTED, "r");
+     while ((mnt_ent = getmntent(mnt_list))) {
+         if (!strcmp(mnt_ent->mnt_dir, mntDir)) {
```

Comment:
the {  can be formatted to goto a new line and the indentation can be fixed for both the if loop and the while loop.

Reply:
Will modify as suggested.